dog_lover/components/topBar.tsx

```diff
@@ -0,0 +1,262 @@
+          boxShadow: "none",
+          width: "100%",
+          height: "100vh",
+          padding: "10px",
+        }}
+      >
+        <Grid
+          container
+          direction="row"
+          spacing={0.2}
+          justifyContent="space-between"
+          alignItems="center"
+        >
+          <Grid item xs={1}>
+            <Box
+              sx={{
+                background: "rgba(103, 164, 255, 0.7)",
+                boxShadow: "none",
+                borderRadius: "30px",
+                padding: "5px",
+                border: "1px solid black",
+                flexGrow: 3,
+              }}
+            >
+              <Typography
+                variant="h5"
+                component="h5"
+                align="center"
+                noWrap
+                sx={{
+                  color: "#EEEBA6",
+                  display: { xs: "none", xl: "block" },
+                }}
+              >
+                DogLover
+              </Typography>
+              <Typography
+                variant="h5"
+                component="h5"
+                align="center"
+                noWrap
+                sx={{
+                  color: "#EEEBA6",
+                  display: { lg: "block", xl: "none" },
+                }}
+              >
+                DL
+              </Typography>
+            </Box>
+          </Grid>
+          <Grid item xs={11}>
+            <Grid
+              container
+              direction="row"
+              spacing={0.3}
+              justifyContent="flex-end"
+              alignItems="center"
+              sx={{ flexGrow: 3 }}
+            >
+              <Grid item>
+                <Button
+                  variant="contained"
+                  size="small"
+                  disableElevation
+                  onClick={twoClicksRandomDog}
+                  sx={{
+                    boxShadow: "none",
+                    background: "rgba(103, 164, 255, 0.7)",
+                    padding: "7px",
+                    border: "1px solid black",
+                    "&:hover": {
+                      backgroundColor: "#7964FC",
+                      boxShadow: "none",
+                    },
+                  }}
+                >
+                  <Typography
+                    variant="subtitle2"
+                    sx={{ textTransform: "Capitalize", color: "#EEEBA6" }}
+                  >
+                    Random Dog Alert
+                  </Typography>
+                </Button>
+              </Grid>
+              <Grid item>
+                <Button
+                  variant="contained"
+                  size="small"
+                  disableElevation
+                  onClick={twoClicksBreedsList}
+                  sx={{
+                    boxShadow: "none",
+                    background: "rgba(103, 164, 255, 0.7)",
+                    padding: "7px",
+                    border: "1px solid black",
```

Comment:
Some of the styles here are repeated. Could you have implemented this some other way?

Reply:
YES definitely with classes (AKA classNames in React). Would look much easier on the eye as well!